src/Components/Books/Book.js

```diff
@@ -1,48 +1,81 @@
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import Proptypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { v4 as uuidv4 } from 'uuid';
-import PropTypes from 'prop-types';
-import { useState, useRef } from 'react';
-import { addBook } from '../../redux/books/books';
-
-const Button = ({ text, handleClick }) => (
-  <button type="button" onClick={handleClick}>{text}</button>
-);
-
-export default function InputBook() {
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
-  const titleInput = useRef(null);
+import { del } from '../../redux/books/books';
+import './Book.css';
 
+export default function Book({ book }) {
   const dispatch = useDispatch();
-
-  const submitBook = () => {
-    const newBook = {
-      id: uuidv4(),
-      title,
-      author,
-    };
-    const res = () => title !== '' && author !== '' && dispatch(addBook(newBook));
-    res();
-    titleInput.current.focus();
-    setTitle('');
-    setAuthor('');
+  const removeBook = (event, bookId) => {
+    event.preventDefault();
+    dispatch(del(bookId));
   };
-
+  const percentage = 12;
   return (
-    <section>
-      <div>
-        <h2>ADD NEW BOOK</h2>
+    <div className="bookCard" id={book.id}>
+      <div className="book_container_left">
+        <span className="bookInfo">
+          <p className="category">{book.category}</p>
+          <p className="title">{book.title}</p>
+          <p className="author">{book.author}</p>
+        </span>
+        <ul className="btn_left_container">
+          <li><button className="btn comment" type="button">Comments</button></li>
+          <span className="bar"></span>
+          <li><button className="btn remove" type="button" onClick={(event) => removeBook(event, book.item_id)}>Remove</button></li>
+          <span className="bar"></span>
+          <li><button className="btn add" type="button">Edit</button></li>
+        </ul>
       </div>
-      <div style={{ display: 'flex', alignItems: 'center' }}>
-        <input type="text" ref={titleInput} placeholder="Book Title" value={title} onChange={(e) => setTitle(e.target.value)} />
-        <input type="text" placeholder="Author" value={author} onChange={(e) => setAuthor(e.target.value)} />
-        <Button text="ADD_BOOK" handleClick={submitBook} />
+      <div className="progress_container">
+        <div className="progressBar">
+          <CircularProgressbar
+            value={percentage}
+            styles={{
+              strokeWidth: 1,
+              root: {},
+              path: {
+                stroke: `rgba(62, 152, 199, ${percentage / 100})`,
+                strokeLinecap: 'butt',
+                transition: 'stroke-dashoffset 0.5s ease 0s',
+                transform: 'rotate(0.25turn)',
+                transformOrigin: 'center center',
+              },
```

Comment:
- [ ] Kindly make the progress bar identical to that used in the Zeplin design. The color of the progress bar in the template design and that of yours are not identical. 

### Zeplin Template

![image](https://user-images.githubusercontent.com/77106962/174446090-449f177e-be2a-435e-9d07-dc12df075789.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/174446106-70949f58-8b1b-4733-9555-f105e54377fb.png)

Reply:
> Hi @stanleySimeon kindly fix linter errors on your pr 👍🏿

I removed the node_modules packages it cannot be installed because of my poor connection
So thank you
No problem